alf/utils/data_buffer.py

```diff
@@ -207,7 +206,13 @@ def _enqueue(self, batch, env_ids=None):
 
         Args:
             batch (Tensor): shape should be
-                ``[batch_size] + tensor_space.shape``
+                ``[batch_size] + tensor_space.shape``.
+                If we ever want to enqueue ``n`` batches (or
+                ``[batch_size, n]`` shaped batch), we'll need to use something
+                like this:
+                ``buf.index_put_((env_ids.reshape(batch_size, 1).repeat(1,
+                batch_size), _current_pos[env_ids] - n to _current_pos[env_ids]
+                ), batch)``
```

Comment:
What does a user need to do to enqueue n batches? How does he access buf?

Reply:
I don't quite understand the code here. Is it valid python code?

Reply:
Removed.  __setitem__((x,y)) can achieve this.